plugins/apache/process-compose.yaml

```diff
@@ -5,6 +5,11 @@ processes:
     command: "echo \"Apache starting on port $HTTPD_PORT\ http://localhost:$HTTPD_PORT\" && apachectl start -f $HTTPD_CONFDIR/httpd.conf -D FOREGROUND"
     availability:
       restart: "always"
+    depends_on:
+      apache-error:
+        condition: process_started
+      apache-access:
+        condition: process_started
```

Comment:
This will start them when apache is started, but I don't think it will stop them when apache stops. I'm not sure if there's a way to have the dependent services terminate with process-compose.